### DISCUSSION TITLE
Download error

### DISCUSSION THREAD

**Discussion by Exodia278** [aclap-dev/video-downloadhelper#633] [Help and Q&A]
### Your question

Hello, I found this error, weird because it was working well some time ago on the same website

{
  "version": "9.0.1.22a1",
  "target": "mozilla",
  "channel": "beta",
  "page_url": "https://live.fc2.com/30281013/",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 00000232d29aa980] Error opening input: Invalid data found when processing input\r\nError opening input file https://us-west-1-media.live.fc2.com/a/stream/30281013/32/playlist?c=LBT0VTdDkBmfYvpVm8Lko&d=izvKVy9_clOKzIin7YHm1VK0LbNEoGdGneEq6Wl2zhsGyDhFE3mzZ6s4OOdDtg2Tq22E_k7AX8s029k2Q9kR8_W_-G7PaYlmBi_UwC35kNNK3p0m0cVSKjv3wCYgJ9ik.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_1af9cfbd-b1ad-4d55-9150-bb30146b0226",
    "downloadable_id": "downloadable_895730261715395"
  }
}

### Addon & CoApp version and details

version: 9.0.1.22a1
target: mozilla
channel: beta
lang: it
coapp: {"found":true,"path":"d:\\Programs\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\Utente1\\dwhelper",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 5
}

**Comment by GrampaWildWilly** [comment]
Somebody else has already reported a problem on this site.  Read this: #509.
